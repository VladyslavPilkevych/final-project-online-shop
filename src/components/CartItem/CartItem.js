import React from 'react';

import styles from './CartItem.module.scss';

function CartItem() {
  return (
    <div>
      <div className={styles.cartWrapper}>
        <div>
          <div>s</div>
          <div>s</div>
          <div>s</div>
          <div>s</div>
        </div>
      </div>
    </div>
  );
}

export default CartItem;
