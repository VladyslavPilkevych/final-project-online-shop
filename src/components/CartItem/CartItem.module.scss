@import '../../assets/styles/global.module.scss';