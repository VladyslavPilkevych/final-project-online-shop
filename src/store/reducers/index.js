import { combineReducers } from 'redux';
import preloaderReducer from './preloaderReducer';
import productsReducer from './productsReducer';
import menuReducer from './menuReducer';
import userReducer from './userReducer';
import searchReducer from './searchReducer';
import cartReducer from './cartReducer';
import miniMenuReducer from './miniMenuReducer';
import filtersCategoriesReducer from './filtersCategoriesReducer';

const reducer = combineReducers({
  loader: preloaderReducer,
  products: productsReducer,
  menu: menuReducer,
  miniMenu: miniMenuReducer,
  filtersCategories: filtersCategoriesReducer,
  user: userReducer,
  search: searchReducer,
  cart: cartReducer,
});

export default reducer;
