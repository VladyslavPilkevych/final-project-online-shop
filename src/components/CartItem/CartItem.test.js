import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import CartItem from './CartItem';
import store from '../../store';

function Component() {
  return (
    <BrowserRouter>
      <Provider store={store}>
        <CartItem />
      </Provider>
    </BrowserRouter>
  );
}

describe('Snapshot CartItem should render', () => {
  test('Snapshot CartItem is rendered', () => {
    const { asFragment } = render(<Component />);
    expect(asFragment()).toMatchSnapshot();
  });
});
