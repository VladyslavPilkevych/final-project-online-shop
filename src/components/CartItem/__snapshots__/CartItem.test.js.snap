// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot CartItem should render Snapshot CartItem is rendered 1`] = `
<DocumentFragment>
  <div>
    <div
      class="cartItemContainer"
    >
      <ul />
    </div>
  </div>
</DocumentFragment>
`;
